rep/processors/CtGovPdfProcessor.py

```diff
@@ -16,13 +17,23 @@ def __init__(self):
         self.VOTE_FOR_PATTERN = r"(Vote for )(.*?)( Seq)"
         self.UNKOWN_PATTERN = r"[a-zA-Z .]"
 
-    def process_blob(self, blob):
+    def process_blob(self, blob, sourceUrl):
         try: 
-            self._process_blob_wrapper(blob)
-        except Exception:
-            raise PdfProcessorException("Unable to process pdf")
+            self._process_blob_wrapper(blob, sourceUrl)
+        except IndexError:
+            raise PdfProcessorException("Unable to process -- PDF does not contain vote data")
+
+    def mark_processed(self, sourceUrl):
+        t = (sourceUrl,)
```

Comment:
can we use descriptive variable names, instead of `t`, `c`?

Reply:
Yup, I lifted that from StackOverflow and forgot to change it, I'll do that

Reply:
I just realized I stole "c" as a variable name from what you wrote in VoteObjectDao — do you want me to change it there too?